src/views/home/home.view.tsx

```diff
@@ -0,0 +1,470 @@
+import React from "react";
+import { connect } from "react-redux";
+import { useTheme } from "react-jss";
+import { push } from "connected-react-router";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+
+import useHomeStyles from "src/views/home/home.styles";
+import ReportIssueButton from "src/views/home/components/report-issue-button/report-issue-button.view";
+import PendingDepositList from "src/views/home/components/pending-deposit-list/pending-deposit-list.view";
+import FiatAmount from "src/views/shared/fiat-amount/fiat-amount.view";
+import AccountList from "src/views/shared/account-list/account-list.view";
+import Spinner from "src/views/shared/spinner/spinner.view";
+import Container from "src/views/shared/container/container.view";
+import TransactionActions from "src/views/shared/transaction-actions/transaction-actions.view";
+import ExitCardList from "src/views/shared/exit-card-list/exit-card-list.view";
+import InfiniteScroll from "src/views/shared/infinite-scroll/infinite-scroll.view";
+import Button from "src/views/shared/button/button.view";
+import { changeHeader, openSnackbar } from "src/store/global/global.actions";
+import * as globalThunks from "src/store/global/global.thunks";
+import { GlobalState } from "src/store/global/global.reducer";
+import * as homeThunks from "src/store/home/home.thunks";
+import { HomeState } from "src/store/home/home.reducer";
+import { getPartiallyHiddenHermezAddress } from "src/utils/addresses";
+import { resetState } from "src/store/home/home.actions";
+import { copyToClipboard } from "src/utils/browser";
+import * as storage from "src/utils/storage";
+import { mergeExits } from "src/utils/transactions";
+import { AsyncTask, isAsyncTaskDataAvailable } from "src/utils/types";
+import { Theme } from "src/styles/theme";
+import { AppDispatch, AppState } from "src/store";
+import { AUTO_REFRESH_RATE } from "src/constants";
+//domain
+import {
+  Account,
+  FiatExchangeRates,
+  HermezWallet,
+  isHermezAccount,
+  PendingDeposit,
+  PoolTransaction,
+  TimerWithdraw,
+} from "src/domain/hermez";
+import { EthereumNetwork } from "src/domain/ethereum";
+
+type HomeStateProps = HomeState &
+  Pick<
+    GlobalState,
```

Comment:
In the case you agreed with [this comment](#r769494184), we should change it also here.